rest/utilities_testing.go

```diff
@@ -181,7 +181,7 @@ func (rt *RestTester) Bucket() base.Bucket {
 		// If running in persistent config mode, the database has to be manually created. If the db name is the same as a
 		// past tests db name, a db already exists error could happen if the past tests bucket is still flushing. Prevent this
 		// by setting the group ID as the current test bucket name by default.
-		sc.Bootstrap.ConfigGroupID = testBucket.GetName()
+		sc.Bootstrap.ConfigGroupID = rt.TB.Name()
```

Comment:
The max group ID length in `(sc *StartupConfig) Validate` (done on line 194) is set to 100. This might cause problems if a subtest has a long name then tests will start failing. 

100 characters is a fair few but it could cause problems in the future. Could there be a way to prevent the name being over 100 characters if it is (such as using a hash maybe)?

Reply:
The good news is that I tested that it errors here, but I agree I should change it.